qutebrowser/browser/webengine/webenginetab.py

```diff
@@ -411,18 +411,34 @@ class WebEngineHistory(browsertab.AbstractHistory):
     def current_idx(self):
         return self._history.currentItemIndex()
 
-    def back(self):
-        self._history.back()
+    def back(self, count):
```

Comment:
Good point about the back/forward methods in browsertab, thanks.

@The-Compiler: what do you think about moving back/forward into AbstractHistory?